services/harmony/app/backends/service-response.ts

```diff
@@ -126,7 +126,8 @@ export async function responseHandler(req: Request, res: Response): Promise<void
 
   const trx = await db.transaction();
 
-  const { job } = await Job.byRequestId(trx, requestId);
+  // TODO - This might be a performance problem for large requests - who is calling this and can we change it to not load all the links?
```

Comment:
Please add a ticket and link to it in this TODO

Reply:
Figured it out and removed.